src/liboslexec/pointcloud.cpp

```diff
@@ -41,8 +41,7 @@ namespace { // anon
 
 #if USE_PARTIO
 
-class PointCloud {
-public:
+struct PointCloud {
```

Comment:
Generally speaking, I use struct only if it's something that could be done in a C struct -- a collection of publicly-accessible data fields that clients are expected to access directly (and, optionally, simple constructors/destructors).

Once it starts looking OO -- anything private, inheritance, methods other than ctr/dtr, etc. -- then I usually call it a class.

Maybe that's old school thinking? I suppose in C++ the only real difference is that it's default public rather than default private?